site/content/docs/5.2/forms/range.md

```diff
@@ -42,6 +47,51 @@ By default, range inputs "snap" to integer values. To change this, you can speci
 <input type="range" class="form-range" min="0" max="5" step="0.5" id="customRange3">
 {{< /example >}}
 
+## Via javascript
+
+{{< added-in "5.3.0" >}}
```

Comment:
The feature is not meant to be bundled in Bootstrap, so this badge could be misleading.